docs/contributing/hacking.rst

```diff
@@ -36,41 +69,108 @@ docker container. To do that, run
     $ ./make.bash --no-clean
     $ GOOS=linux GOARCH=amd64 ./make.bash --no-clean
 
-After that, you should be able to compile Deis' components as normal.
+Once you can compile to ``linux/amd64``, you should be able to compile Deis'
+components as normal.
+
+Configuring Boot2Docker (Mac)
+`````````````````````````````
+
+Deis needs a Docker registry running independently of the Deis cluster. On
+OS X, you will need to run this docker registry inside of Boot2Docker (http://boot2docker.io).
+
+Install Boot2Docker according to the normal installation instructions. When you
+run ``init``, we highly recommend allocating a large disk, since the Docker
+registry that will live there is fairly large.
+
+.. code-block:: console
+
+    $ boot2docker init --disksize=100000
+
+That will create virtual disk that can eventually take up a full 100,000MB of
+disk space. Then start up Boot2Docker.
+
+Once you have run ``boot2docker up``, you should be able to connect to it. You
+need to make a minor editor to the boot2docker config:
+
+.. code-block:: console
+
+    $ boot2docker ssh sudo vi /var/lib/boot2docker/profile
+
+Inside of the profile, you need to add one line:
+
+.. code-block:: console
+
+    EXTRA_ARGS="--insecure-registry 192.168.59.103:5000"
```

Comment:
This is the IP every time I've launched b2d/the registry, but I don't think it's explicit anywhere. May be good to do a sanity check with `boot2docker ip`.